datasets/loopfietsnetwerk/dataset.json

```diff
@@ -165,6 +165,10 @@
               "type": "string",
                "description": "Typering point of interest"
            },
+            "bkPoi": {
```

Comment:
We streven er altijd naar om de namen zo duidelijk mogelijk te laten zijn. Wellicht is `bkPoi` al te cryptisch? Zou het mogelijk zijn een iets langere naam te gebruiken?  (overigens kun je dan in PostgreSQL wel een korte kolomnaam gebruiken door een `shortname` toe te voegen). Zie: https://schemas.data.amsterdam.nl/docs/ams-schema-spec.html#naamgeving